File: C#BasicsMarch2015/4.ComplexConditions/FruitOrVegetable/FruitOrVegetable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FruitOrVegetable
{
    class FruitOrVegetable
    {
        static void Main(string[] args)
        {
            Console.Write("Enter fruit or vegetable: ");
            var kind = Console.ReadLine().ToLower();
            if(kind == "banana" || kind == "apple" || kind == "kiwi" || kind == "cherry" || kind == "lemon" || kind == "grapes")
            {
                Console.WriteLine("Fruit");
            }
            else if(kind == "tomato" || kind == "cucumber" || kind == "pepper" || kind == "carrot")
            {
                Console.WriteLine("Vegetable");
            }
            else
            {
                Console.WriteLine("Unknown");
            }
        }
    }
}

File: JSFundamentalsMay2016/02.ControlFlowLogic/FruitOrVegetable.js
function fruitOrVegetable(word) {
    let fruit = ['banana', 'apple', 'kiwi', 'cherry', 'lemon', 'grapes', 'peach'];
    let vegetables = ['tomato', 'cucumber', 'pepper', 'onion', 'garlic', 'parsley'];

    if(fruit.includes(word)){
        console.log('fruit');
    } else if(vegetables.includes(word)){
        console.log('vegetable');
    } else {
        console.log('unknown');
    }
}

fruitOrVegetable('apple');